packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -247,48 +253,64 @@ export class RestAPIClient {
 	}
 
 	/**
-	 * Deletes a site.
+	 * Deletes a site belonging to the user.
+	 *
+	 * This method will perform a quick check to ensure the target site
+	 * belongs to the user.
 	 *
-	 * If the target site has an upgrade purchased within the
+	 * Additionally, if the target site has an upgrade purchased within the
 	 * sandboxed environment, it can be deleted without first
 	 * cancelling the subscription.
 	 *
 	 * Otherwise the active subscription must be first cancelled
-	 * otherwise the REST API will throw a HTTP 403 status.
+	 * or else the REST API will throw a HTTP 403 status.
 	 *
 	 * @param {SiteDetails} expectedSiteDetails Expected details for the site to be deleted.
```

Comment:
**Nit (optional)**: I would update the JSDOC comment here to match the new signature.

Reply:
Thanks, done in 5e1aaadf55006cb8a4f58a8d5a3570a08c306b3c.